go/service/identify.go

```diff
@@ -147,8 +149,234 @@ func (h *IdentifyHandler) resolveUserOrTeam(ctx context.Context, arg string) (u
 	return res.UserOrTeam(), nil
 }
 
+func (h *IdentifyHandler) resolveUser(ctx context.Context, assertion string) (u keybase1.User, err error) {
+
+	res := h.G().Resolver.ResolveFullExpressionNeedUsername(ctx, assertion)
+	err = res.GetError()
+	if err != nil {
+		return u, err
+	}
+	u = res.User()
+	if !u.Uid.Exists() {
+		return u, fmt.Errorf("no resolution for: %v", assertion)
+	}
+	return u, nil
+}
+
 func (h *IdentifyHandler) ResolveIdentifyImplicitTeam(ctx context.Context, arg keybase1.ResolveIdentifyImplicitTeamArg) (res keybase1.ResolveIdentifyImplicitTeamRes, err error) {
-	return res, fmt.Errorf("TODO: implement ResolveIdentifyImplicitTeam")
+	ctx = libkb.WithLogTag(ctx, "RII")
+	defer h.G().CTrace(ctx, "IdentifyHandler#ResolveIdentifyImplicitTeam", func() error { return err })()
+
+	h.G().Log.CDebugf(ctx, "ResolveIdentifyImplicitTeam assertions:'%v'", arg.Assertions)
+
+	if len(arg.Assertions) == 0 {
+		return res, fmt.Errorf("empty assertions")
+	}
+
+	split := strings.Split(arg.Assertions, "#")
```

Comment:
Can we bump this down to a helper or utility function?  BTW, I saw it this morning, there is already some code here in `kbfs/kbfs.go`, hopefully that's not totally duplicated by this.